README.md

```
@@ -31,11 +31,11 @@
   <img src="https://github.com/upscayl/upscayl/assets/25067102/6287fd40-2c91-4028-b1d6-3986e77d8211" width="200px" />
 </a>-->
 
-<a href="https://t.me/iamnayam">
+<a href="https://t.me/iamnayam">target="_blank"
   <img src="https://user-images.githubusercontent.com/25067102/209297095-a3db856f-b760-40bb-a68e-f3a3086e18c7.png" width="200px" />   
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)